### ISSUE TITLE
Clarify needed permissions

### ISSUE THREAD

**Issue by tiagonmas** [jstrieb/github-stats#81]
Current instructions state on step 2: "Personal access token must have permissions: **read:user** and **repo**."
When we are creating a New fine-grained personal access token , we have [many permissions](https://docs.github.com/en/rest/overview/permissions-required-for-fine-grained-personal-access-tokens) to chose from. It's not clear what permissions need to be added for this to work. There is no read:user or repo.

**Comment by MadRoosterCro** [comment]
I am not sure what are all the permissions needed in order to generate the images. Need clarification.

**Comment by ellielle** [comment]
The permissions you are looking for are under Tokens (classic), not the beta fine-grained tokens.

![H7SxIdL 1](https://user-images.githubusercontent.com/40385743/210797027-f543b6a1-5b73-4f9e-9cb7-79cd007d014a.png)
<br>
The correct permissions page can be found at [https://github.com/settings/tokens/new](https://github.com/settings/tokens/new).

**Comment by shadyueh** [comment]
Could you update the repo README instructions? I needed to find this thread to finally fix the same problem

**Comment by py563** [comment]
yes please update the readme with only use tokens(classic). i have the fine-grained token with read repo properties still it generates images with 0 count

**Comment by hebelia** [comment]
im pretty sure it is not safe to grant such permissions to a stranger but .. 😪style points

**Comment by jstrieb** [comment]
Addressed by 6f25f02f82141f3bc35bba4a92feb590e6bbcca9.
